.gitignore

```diff
@@ -1,5 +1,6 @@
 # no generated files in version control
-src/main/gen/
+**/src/main/gen/
```

Comment:
generated code like jabref.gui/src/main/gen/org/jabref/logic/importer/fileformat/mods/RoleTermDefinition.java is put under version control; is this desired?

Reply:
That should not happen. We have to remove that code.

I will double check.